java/arcs/core/host/generated/BUILD

```diff
@@ -0,0 +1,22 @@
+load("//third_party/java/arcs/build_defs:build_defs.bzl", "arcs_kt_plan", "arcs_kt_schema")
+
+licenses(["notice"])
+
+package(default_visibility = ["//visibility:public"])
+
+arcs_kt_schema(
```

Comment:
you can replace these 2 with just one: arcs_kt_gen

Reply:
Done!